tests/switchmap_/server/db/table/test_systemstat.py

```
@@ -0,0 +1,344 @@
+#!/usr/bin/env python3
```

Ruff (0.14.1)

Shebang is present but file is not executable

(EXE001)